Makefile

```diff
@@ -22,6 +22,10 @@ CORE_GENERATOR ?= rocketchip
 # soc generator doesn't actually do anything at all.
 SOC_GENERATOR ?= nop
 
+# The testbench for simulation
+# FIXME: what's the default?
+TESTBENCH ?= none
```

Comment:
This is necessary for setting `STRIP_PATH` in the PrimeTime script(path from the top module to DUT), which is generally the testbench. Otherwise, PrimeTime fails in signal activity annotations. I guess there may be a much better name, though.